File: MODUL.4/PRAK402_2310817120010_NurHikmah.py
batas = int(input("Input\n"))

print("\nOutput")
for i in range(1, batas + 1):
    if i % 2 != 0:
        print(i, end=' ')

print("")
for i in range(batas, 0, -1):
    if i % 2 == 0:
        print(i, end=' ')

File: MODUL.4/PRAK402_2310817120010_NurHikmah.c
#include <stdio.h>

int main()
{
    int batas, i;

    printf("Input\n");
    scanf("%d", &batas);

    printf("\nOutput\n");
    for (i = 1; i <= batas; i++)
    {
        if (i % 2 != 0)
        {
            printf("%d ", i);
        }
    }
    printf("\n");
    for (i = batas; i > 0; i--)
    {
        if (i % 2 == 0)
        {
            printf("%d ", i);
        }
    }
    return 0;
}